Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpMaterialImporter.cpp

```diff
@@ -71,8 +73,8 @@ namespace AZ
 
                     if (matFound == materialMap.end())
                     {
-                        std::shared_ptr<AssImpSDKWrapper::AssImpMaterialWrapper> assImpMaterial =
-                            std::shared_ptr<AssImpSDKWrapper::AssImpMaterialWrapper>(new AssImpSDKWrapper::AssImpMaterialWrapper(context.m_sourceScene.GetAssImpScene()->mMaterials[materialIndex]));
+                        AZStd::shared_ptr<AssImpSDKWrapper::AssImpMaterialWrapper> assImpMaterial =
+                            AZStd::shared_ptr<AssImpSDKWrapper::AssImpMaterialWrapper>(aznew AssImpSDKWrapper::AssImpMaterialWrapper(context.m_sourceScene.GetAssImpScene()->mMaterials[materialIndex]));
```

Comment:
Was this complaining about aznew?
aznews supposed to be used for classes that have a AZ_CLASS_ALLOCATOR tag in them, and AssImpMaterialWrapper does not...

Reply:
No complaint at least on windows, but I'll revert it to new anyway.